src/js/core/read_write_proxy.js

```diff
@@ -317,7 +318,7 @@ export class ReadWriteProxy {
         if (!data) {
             return ExplainedResult.bad("Data is empty");
         }
-        if (!Number.isInteger(data.version) || data.version < 0) {
+        if (!savegameInterfaces[data.version] && (!Number.isInteger(data.version) || data.version < 0)) {

```

Comment:
shouldn't this be an `||`? am I missing something here?

Reply:
It also supports string for savegame interfaces and if its not in the savegameInterfaces it doesn't exists. Maybe i need to remove the number check

Reply:
Removed number check